schema/configuration-structure.asn

```diff
@@ -37,12 +37,7 @@ BEGIN
    }
 
    Hart ::= SEQUENCE {
-      hartid CHOICE { /* 4 choices use 2-bit as the index*/
-        hartId4  FlexibleRange2, /* For hart number [1..4] */
-        hartId8  FlexibleRange3, /* For hart number [1..8] */
-        hartId16 FlexibleRange4, /* For hart number [1..16] */
-        hartId   FlexibleRange   /* For hart number > 16 */
-      },
+      hartId   FlexibleRange,
 
```

Comment:
We can remove the definitions of different sizes of hart ID and only keep "hartId".
However, we can still keep those definitions for the different length integers in CS. People use that when they need it, and of course, they know how to use the correct structure to refer to the values.

Reply:
I'll put them back in.

Reply:
ok. thanks.